src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommender.java

```diff
@@ -0,0 +1,104 @@
+package nearsoft.academy.bigdata.recommendation;
+
+import org.apache.commons.collections.functors.WhileClosure;
+import org.apache.commons.math.stat.descriptive.summary.Product;
+
+import java.io.BufferedReader;
+import java.util.*;
+import java.io.FileReader;
+
+public class MovieRecommender {
+
+
+    ArrayList<Review> listaDeReviews = new ArrayList<>();
```

Comment:
This attribute has "Default" access modified, is this necessary? if not lets change it to private

Reply:
Lets avoid writing code in "spanglish", please rename the variable names to english